client/src/views/ProfilePageComponents/ThankyouCard.js

```diff
@@ -5,13 +5,23 @@ import { motion } from "framer-motion";
 import { FaCheckCircle, FaEdit } from "react-icons/fa";
 import { useSetRecoilState } from 'recoil';
 import { isSubmittedState } from '../../store/atoms/userProfileSate';
+import axios from 'axios';
+
+const BACKEND_URL = process.env.REACT_APP_BACKEND_URL;
 
 export default function ThankyouCard() {
   const setIsSubmitted = useSetRecoilState(isSubmittedState);
 
   const handleEditProfile = () => {
     setIsSubmitted(false);
     localStorage.setItem('isSubmitted', 'false');
+    axios.put(`${BACKEND_URL}/api/auth/update`, {
+      isProfileSubmitted: false,
+    }, {
+      headers: {
+        Authorization: `Bearer ${localStorage.getItem("authToken")}`,
```

Comment:
Utilize useAuth() wherever possible instead of fetching from local storage everytime.